<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ownage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gameName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>491</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>gameName</string>
    </property>
   </widget>
   <widget class="QLabel" name="gamePath">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>491</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>gamePath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openGamePathButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>open in file manager</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameId">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>gameId</string>
    </property>
   </widget>
   <widget class="QLabel" name="crackableLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>crackable:</string>
    </property>
   </widget>
   <widget class="QLabel" name="crackableStatus">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
   </widget>
   <widget class="QLabel" name="crackedLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cracked:</string>
    </property>
   </widget>
   <widget class="QLabel" name="crackedStatus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>false</string>
    </property>
   </widget>
   <widget class="QLabel" name="apiLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>api type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="apiStatus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>unkown</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionabout_ownage_2"/>
   </widget>
   <addaction name="menuabout"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionabout_ownage">
   <property name="text">
    <string>about ownage</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionabout_ownage_2">
   <property name="text">
    <string>about ownage</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
